generator/src/Command/ScrapeCommand.php

```diff
@@ -0,0 +1,235 @@
+<?php
+declare(strict_types=1);
+
+namespace MongoDB\CodeGenerator\Command;
+
+use InvalidArgumentException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Component\Yaml\Yaml;
+
+use function array_column;
+use function array_combine;
+use function array_key_exists;
+use function array_shift;
+use function assert;
+use function explode;
+use function file_exists;
+use function file_get_contents;
+use function file_put_contents;
+use function htmlspecialchars_decode;
+use function ksort;
+use function mkdir;
+use function sort;
+use function str_contains;
+use function str_ends_with;
+use function str_replace;
+use function strip_tags;
+use function trim;
+use function var_export;
+
+use const PHP_EOL;
+
+final class ScrapeCommand extends Command
+{
+    private Crawler $crawler;
+
+    /** @var array<string, string> $tabs Associative array of names to table ids */
+    private array $tabs;
+
+    public function __construct(
+        private string $configDir,
+    ) {
+        parent::__construct();
+    }
+
+    public function configure(): void
+    {
+        $this->setName('scrape');
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $index = file_get_contents('https://docs.google.com/spreadsheets/d/e/2PACX-1vROpGTJGXAKf2SVuSZaw16NwMVtzMVGH9b-YiMtddgZRZOjOO6jK2YLbTUZ0N_qe74nxGY9hYhUe-l2/pubhtml');
```

Comment:
I published the Google Sheet publicly as an HTML document because csv and tsv doesn't have correct support for newline. Also, I can edit in Google and run my command without all the hassle of Google authentication.